### PR TITLE
Update document for 2020.

### PR DESCRIPTION
**Pull request by ktlim**

### CODE REVIEW

File: LDM-148.tex [hunk 1]
```diff
@@ -444,6 +444,7 @@ \subsubsection{Archiving}\label{archiving}
 including the wavefront sensors and the guide sensors of the LSSTCam or
 ComCam when so configured, retrieving them from their respective Camera
 Data System instances.
+The Image Archiver service for at least LSSTCam will be implemented using the Camera Control System's image writing component as a result of the removal of crosstalk-corrected images.
```
Comment (jdswinbank) [inline LDM-148.tex:446]: The “as a result...” is just capturing historical decision making processes; I'm not sure it adds anything to this document, and, as such, I would remove it.

File: sizing.tex [hunk 2]
```diff
@@ -1,87 +1,19 @@
...
-\begin{figure}
-\centering
-\includegraphics[width=\textwidth]{images/SizingModel.pdf}
-\caption{DMS Infrastructure Sizing and Estimation.}
-\label{fig:sizing-model}
-% Silently cite documents mentioned in diagram
-\nocite{LDM-148,LDM-151,LDM-152,LDM-135,LDM-138,LDM-140,LDM-129}
-\nocite{LSE-81,LSE-82,LDM-141,LDM-139,LDM-144,LDM-143}
-\nocite{LPM-17,LSE-29,LSE-30,LSE-61,LSE-163}
-\nocite{Document-5373,LDM-142,LSE-78}
```
Comment (jdswinbank) [inline sizing.tex:25]: How many of these documents should be formally deprecated?

(Not necessarily on this ticket, but this is a good opportunity to note them).

File: LDM-148.tex [hunk 3]
```diff
@@ -1138,7 +1138,7 @@ \subsubsection{Identity Management}\label{identity-management}
 
 \subsubsection{ITC Provisioning and Management}\label{itc-provisioning-management}
 
-This service provides for acquisition, change and configuration management, and provisioning of information technology components, whether purchased ``bare metal'', provided by the NCSA Commons, or provided by agreements with outside providers.
+This service provides for acquisition, change and configuration management, and provisioning of information technology components, whether purchased ``bare metal'', provided by a "commons" at the US Data Facility, or provided by agreements with outside providers.
```
Comment (jdswinbank) [inline LDM-148.tex:1147]: The term “commons” here feels awkward. Perhaps “shared with multiple projects at the US Data Facility”, or similar? Or even “...held in common with..”?

File: LDM-148.tex [hunk 4]
```diff
@@ -545,7 +545,7 @@ \subsection{Interfaces}\label{base-interfaces}
 from the control service to the batch service, whether to the Commissioning Cluster or over the international network to the Offline Production Enclave, and return status and
 result information.
 
-Telemetry Gateway from Prompt US Enclave services: RabbitMQ is
+Telemetry Gateway from Prompt US Enclave services: RabbitMQ or Kafka are
```
Comment (jdswinbank) [inline LDM-148.tex:548]: Do we know which? I'm not sure if this “or” expresses a “TBD” or a “we use one one or the other, depending on the type of data to be transferred”.

Comment (ktlim) [reply LDM-148.tex:548]: It's a TBD.  I could replace with a generic "messaging system", explicitly label "(TBD)", or leave as is.  Preference?

File: sizing.tex (hunk 2 above)
Comment (ktlim) [reply sizing.tex:25]: I guess LDM-141, 139, 144, 143 (and 138/140) could be demoted to "Document" status.  They're still valuable, but they no longer control the design of the system.

File: LDM-148.tex (hunk 4 above)
Comment (ktlim) [reply LDM-148.tex:548]: I went with "(TBD)".

File: sizing.tex (hunk 2 above)
Comment (ktlim) [reply sizing.tex:25]: Submitted RFC to capture this.

**Comment by ktlim** [comment]
Somehow this PR seems stuck.  I will close it and create a new one.

**Pull request closed without merging**